crates/hdk3/src/entry.rs

```diff
@@ -1,4 +1,605 @@
-pub mod create_entry;
-pub mod delete_entry;
-pub mod hash_entry;
-pub mod update_entry;
+use crate::prelude::*;
+
+/// General function that can create any entry type.
+///
+/// This is used under the hood by `create_entry`, `create_cap_grant` and `create_cap_claim`.
+///
+/// The host builds a `Create` header for the passed entry value and commits a new element to the
+/// chain.
+///
+/// Usually you don't need to use this function directly; it is the most general way to create an
+/// entry and standardises the internals of higher level create functions.
+///
+/// @see create_entry
+/// @see create_cap_grant
+/// @see create_cap_claim
```

Comment:
```suggestion
### See
/// [`create_entry`]
/// [`create_cap_grant`]
/// [`create_cap_claim`]
```
I'm not sure if your browser has some plugin for `@` but it just shows as raw text for me?